### ISSUE TITLE
Kodi 19 - Python 3

### ISSUE THREAD

**Issue by d4vex** [markop159/KODI-Popcorn-Time#136]
Any chance to update / futurize the plugin for the new Kodi release?

**Comment by giaur500** [comment]
I think there's no chance, as this addon is actually abandoned. So, RIP. And good night.

**Comment by S0und** [comment]
Just get this https://www.stremio.com/

**Comment by markop159** [comment]
Working on new project if anyone wants to try, currently in alpha stage, not all functionalities implemented yet, but it works on Kodi 19: [plugi.video.flixtor](https://github.com/markop159/plugin.video.flixtor)

**Comment by lubo** [comment]
I've made the add-on compatible with Kodi 19 in https://github.com/lubo/KODI-Popcorn-Time. I'd appreciate some feedback and I might release a new version later.

**Comment by ruipisco** [comment]
Hi @lubo i've tested on windows and it work's fine, but on macos the torrents don´t load because it supose it uses the darwin version of torrent2http. Does anybody know if it work on linux and raspberry pi?

**Comment by lubo** [comment]
@ruipisco I use it on Arch Linux on x86_64 and on LibreELEC on RPi 4. I won't reply to further inquiries about my fork here, as this repository is not concerned with my fork in any way.

**Comment by giaur500** [comment]
Definitely. Your fork works just fine (I use CoreElec), but any further discussion should go there not here. Flixtor works fine too, but it includes much less movies and worse quality

**Comment by matthuo333** [comment]
For TV Box with android 10 and KODI 19.3, it is not available regardless of Fixtor or Lubo's
